common/flatpak-run.c

```diff
@@ -3418,9 +3418,18 @@ check_parental_controls (const char     *app_ref,
   if (app_ref_parts == NULL)
     return FALSE;
 
-  system_bus = g_bus_get_sync (G_BUS_TYPE_SYSTEM, NULL, error);
+  /* If we don't have a system bus then we certainly don't have
+   * parental controls. If this was a real security mechanism, we'd have
+   * to fail closed here (because it's possible to tell 'flatpak run'
+   * a wrong system bus address), but malcontent is already possible
+   * to bypass. */
+  system_bus = g_bus_get_sync (G_BUS_TYPE_SYSTEM, NULL, NULL);
```

Comment:
Yeah, [not sure about this one](https://github.com/flatpak/flatpak/issues/3902#issuecomment-708691401).

Reply:
I've dropped that part for now.